src/utils/catalogue.js

```diff
@@ -72,20 +72,32 @@ export async function fetchOfferings (query, currentPage) {
   return fetchFromCatalogue(sparQLQuery)
 }
 
-function getSparQLOfferingsCountQueryString () {
-  const baseString = `
+function getSparQLOfferingsCountQueryString (query) {
+  if (query !== undefined) {
+    const baseString = `
+    ${prefixes}
+
+    SELECT DISTINCT (COUNT(?offering) as ?count)
+    WHERE {
+      ${getOfferingQueryFilter(query)}
+    }
+  `
+    return encodeURI(baseString)
+  } else {
+    const baseString = `
     ${prefixes}
 
     SELECT DISTINCT (COUNT(?offering) as ?count)
     WHERE {
       ?offering a sedi:Offering .
     }
   `
-  return encodeURI(baseString)
+    return encodeURI(baseString)
+  }
 }
```

Comment:
Hint: sorry if my previous comment wasn't clear. I meant that you can leverage the fact that if query is an empty string, then it will match all offerings. So you don't need to resort to an `if` statement to distinguish the filtered case from the unfiltered one. You can do simply something like:
```suggestion
function getSparQLOfferingsCountQueryString (query = "") {
  const baseString = `
  ${prefixes}

  SELECT DISTINCT (COUNT(?offering) as ?count)
  WHERE {
    ${getOfferingQueryFilter(query)}
  }
  `
  return encodeURI(baseString)
}
```

Reply:
Ahh I thought that the:
```
    WHERE {
      ?offering a sedi:Offering .
    }
```

was required for offerings